x/ecocredit/client/tx.go

```diff
@@ -29,6 +29,7 @@ const (
 	FlagRemoveIssuers string = "remove-issuers"
 	FlagReferenceId   string = "reference-id"
 	FlagIssuer        string = "issuer"
+	FlagRetireTo      string = "retire-to"
```

Comment:
this was my bad, i should've updated the issue to use the jurisdiction terminology 🤦🏻 

also, we should add some docs to `Long` specifying that adding `--retirement-jurisdiction` flag will cause the credits to be retired to the `to` address, otherwise they will be sent as tradable

Reply:
👍 done and done!